pkgs/games/arena/default.nix

```diff
@@ -68,6 +68,8 @@ stdenv.mkDerivation rec {
       chess board & DGT clocks and much more.
     '';
     license = lib.licenses.unfree;
+    sourceProvenance = with lib.sourceTypes; [ binaryNativeCode ];
+    homepage = "http://www.playwitharena.de";
     platforms = ["x86_64-linux"];
     hydraPlatforms = lib.platforms.none;
```

Comment:
Please also remove this. Hydra doesn't build unfree packages so this is not needed.

Reply:
Done